python_modules/dagster/dagster/__init__.py

```diff
@@ -330,7 +330,9 @@
     make_values_resource as make_values_resource,
     resource as resource,
 )
-from dagster._core.definitions.result import MaterializeResult as MaterializeResult
+from dagster._core.definitions.result import (
+    MaterializeResult as MaterializeResult,
```

Comment:
Should `ObserveResult` be added here?

Reply:
We decided to leave it private for the immediate future